### PR TITLE
VB-2570 DEV-Orchestration service, Update SQS/SNS queue updates IRSA

### PR DESCRIPTION
**Pull request by AledWEvans**
Updates to helm values files to allow dev to use IRSA config and leave other environments using old approach.

### CODE REVIEW

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -16,6 +16,11 @@ generic-service:
     PRISON-REGISTER_API_URL: https://prison-register-dev.hmpps.service.justice.gov.uk
     WHEREABOUTS_API_URL: https://whereabouts-api-dev.service.justice.gov.uk
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
+    HMPPS_SQS_USE_WEB_TOKEN: "true"
+
+  namespace_secrets:
+    HMPPS_SQS_QUEUES_PRISONVISITSEVENTS_ARN: "topic_arn"
```
Comment (dhirajshettymoj) [inline helm_deploy/values-dev.yaml:22]: do we not need the other properties in dev?

File: helm_deploy/values-staging.yaml [hunk 2]
```diff
@@ -17,6 +17,17 @@ generic-service:
     WHEREABOUTS_API_URL: https://whereabouts-api-dev.service.justice.gov.uk
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
 
+  namespace_secrets:
```
Comment (dhirajshettymoj) [inline helm_deploy/values-staging.yaml:20]: will staging and dev end up listening on the same hmpps queue as staging might not be there on the env where the queues are configured?

File: helm_deploy/values-dev.yaml (hunk 1 above)
Comment (AledWEvans) [reply helm_deploy/values-dev.yaml:22]: I'm not sure HMPPS_SQS_QUEUES_PRISONVISITSEVENTS_QUEUE_NAME may be

File: helm_deploy/values-staging.yaml (hunk 2 above)
Comment (AledWEvans) [reply helm_deploy/values-staging.yaml:20]: Yeha the fix for this is in CP!

**Review by dhirajshettymoj — APPROVED** [review]
LGTM. As discussed envs other than dev might need to be revisited once we get it running on dev.

**Pull request merged**